http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -538,59 +505,6 @@ object CommonTestData {
   val wsmResourceId = WsmControlledResourceId(UUID.randomUUID())
```

Comment:
can we also remove wsmResourceId and wsmResourceIdOpt?

Reply:
Yeah - which we should! But not in this PR.